anndata/_core/index.py

```diff
@@ -138,9 +141,14 @@ def make_slice(idx, dimidx, n=2):
     return tuple(mut)
 
 
-def get_vector(adata, k, coldim, idxdim, layer=None):
+def get_vector(
+    adata: "anndata.AnnData",
+    k: str,
+    coldim: Literal["obs", "var"],
+    idxdim: Literal["obs", "var"],
+    layer: Optional[str] = None,
```

Comment:
How about adding an `axis=None` param that defaults to 0 for `obsp` and to ` for `obsm`?

Reply:
To resolve the ambiguity of indexing into `obsp` (defaulting to rows), and potentially allows indexing into rows of `obsm` (where it defaults to columns).

Reply:
To what for `obsm`?